src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -55,7 +55,7 @@ void setUserStore(UserStore userStore) {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
-    request.getRequestDispatcher("/WEB-INF/view/login.jsp").forward(request, response);
```

Comment:
would revert this change since you're just adding a space.

Reply:
Sorry, I'm not totally sure how I would revert that change back, how did I add a space?